containers/ecr-viewer/integration/data/db/schema/core.test.ts

```diff
@@ -0,0 +1,449 @@
+import { dbNamespace, getDb } from "@/app/api/services/database";
+import { getSql } from "@/app/api/services/dialects/common";
+import { getDbUtils } from "@/app/data/db/utils";
+import { sql } from "kysely";
+
+const db = getDb();
+const schema = dbNamespace();
+let utils = getDbUtils();
+
+interface MigrationRow {
+  name: string;
+  timestamp?: string;
+}
+
+describe("Common Schema: ", () => {
+  afterAll(async () => {
+    await db.destroy();
+  });
+
+  it("has a schema", async () => {
+    expect(await utils.schemaExistsByName(db, schema)).toBe(true);
```

Comment:
how are these actually using hte migration? I feel like I'm missing a step. Also can the helpers to create tables and such be replaced with the migrators now?